pages/model/[uuid].tsx

```diff
@@ -168,6 +179,25 @@ function Model() {
     await postEndpoint(`/api/v1/version/${version?._id}/reset-approvals`, {}).then((res) => res.json())
   }
 
+  const handleDelete = () => {
+    setDeleteConfirmContent(<Box />)
+    setDeleteConfirmOpen(true)
+  }
+
+  const handleDeleteCancel = () => {
+    setDeleteConfirmOpen(false)
+  }
+
+  const handleDeleteConfirm = async () => {
+    await deleteEndpoint(`/api/v1/model/${uuid}`, {}).then((res) => {
+      if (res.status < 400) {
```

Comment:
Should be 200

Reply:
As discussed if @a3957273 is happy with it being < 400 we can leave it - although it might have issues in the future if we change the API call to have forms of 2XX responses!

Reply:
Leaving as <400, may possibly change in future pending discussion